ibm_catalog.json

```diff
@@ -32,18 +32,19 @@
       "features": [
         {
           "title": "Watson Discovery instance",
-          "description": "Creates the Watson Discovery Instance. [Learn more](https://cloud.ibm.com/docs/discovery-data?topic=discovery-data-about)"
+          "description": "Provision and configure an instance of IBM Watson Discovery. Explore more about IBM Watson Discovery [here](https://cloud.ibm.com/docs/discovery-data?topic=discovery-data-about)"
         },
         {
-          "title": "Optional Integrations",
-          "description": "This solution can be integrated with [Cloud automation for Observability](https://cloud.ibm.com/catalog/7a4d68b4-cf8b-40cd-a3d1-f49aff526eb3/architecture/deploy-arch-ibm-observability-a3137d28-79e0-479d-8a24-758ebd5a0eab-global) that supports configuring resources for logging, monitoring and activity tracker event routing"
+          "title": "Optional observability tools",
```

Comment:
I'll suggest to use `Observability` or `Observability integration` as title and `optional` can be sued in description.
You can take reference of data/orchestrate for this.